### PR TITLE
Spacefinder fixed on mobile

### PR DESCRIPTION
**Pull request by Calanthe**
thx @mattosborn

### CODE REVIEW

**Comment by uplne** [comment]
Why is this AB test? It's going to the 100% audience so it needs only switch.

**Comment by Calanthe** [comment]
You are right, the audience should be 2% + control.

File: common/app/conf/switches.scala [hunk 1]
```diff
@@ -590,6 +590,11 @@ object Switches {
     safeState = Off, sellByDate = new LocalDate(2015, 5, 27)
   )
 
+  val ABDeferSpacefinder = Switch("A/B Tests", "ab-defer-spacefinder",
+    "A/B test to defer execution of spacefinder until images and richlinks have been loaded.",
+    safeState = Off, sellByDate = new LocalDate(2015, 5, 15)
```
Comment (uplne) [inline common/app/conf/switches.scala]: This will end next Friday. Is it enough time?

**Comment by uplne** [comment]
:+1:

**Pull request merged**